warehouse/internal/repo/upload_test.go

```diff
@@ -140,6 +143,364 @@ func TestUploads_Get(t *testing.T) {
 	})
 }
 
+func TestUploads_GetToProcess(t *testing.T) {
```

Comment:
Some of the tests presented here could have been combined with `TestUploads_Processing`